src/platform/testing/e2e/cypress/support/index.d.ts

```diff
@@ -0,0 +1,110 @@
+/// <reference types="cypress" />
```

Comment:
this file makes it so we will get intellisense when we type `cy.` , and get suggestions for all the methods listed here